pkg/database/token.go

```diff
@@ -472,16 +480,21 @@ func (db *Database) updateStatsTokenInvalid(t time.Time, authApp *AuthorizedApp)
 
 // updateStatsTokenClaimed updates the statistics, increasing the number of
 // tokens claimed.
-func (db *Database) updateStatsTokenClaimed(t time.Time, authApp *AuthorizedApp) {
+func (db *Database) updateStatsTokenClaimed(t time.Time, authApp *AuthorizedApp, isUserReport bool) {
```

Comment:
I always get weary of miscellaneous booleans that control flow. Any ideas on how we could make this better? Maybe accepting a "ReportType" enum?